web/modules/custom/server_general/src/ThemeTrait/PersonCardGridThemeTrait.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\server_general\ThemeTrait;
+
+/**
+ * Helper for a grid of Person Cards.
+ */
+trait PersonCardGridThemeTrait {
+
+  /**
+   * Build a grid of Person cards.
+   *
+   * @param array $items
+   *   Render arrays of Person cards.
+   */
+  protected function buildElementPersonCardGrid(array $items): array {
+    return [
+      '#type' => 'container',
+      '#attributes' => [
```

Comment:
We shouldn't pass CSS from PHP, that's the responsibility of the twig files. We shuld only pass the _intention_